README.md

```diff
@@ -351,10 +351,131 @@ practice for modules to claim their own area in the configuration object is to
+particular, dynamic computed configurations. Computed values can help to
+simplify dependency issues by allowing values to be computed at runtime from a
+function or string template. (Note there is a small cost with computed config
+values which could be important depending on the use case.)
+
+`bedrock.util.Config` creates a wrapper around a config object and optional
+options. A common setup could look like the following.
+
+```js
+// an empty config object
+let config = {};
+// common options
+let options = {
+  // the config
+  config: config
+  // local vars used during string template evaluation
+  locals: config
+};
+// wrap the config
+let c = new bedrock.util.Config(config, options);
+```
+
+Bedrock provides a shared wrapper around the common `bedrock.config` as
+`bedrock.util.config`.
+
+To do simple sets of config data, use the `set()` API.
+
+```js
+let c = bedrock.util.config;
+// set a config variable with a path
+// path components are created as needed
+c.set('server.port', 8443);
+// config is now {"server": {"port": 8443}}
+```
+
+Normal code and the config API can be mixed. A useful helper is `require()`.
+This call lets you simplify ensuring a full object path exists before setting
+data. Objects in the path are created as needed.
+
+```js
+// create container object if needed
+c.require('accounts.admin');
+// the config is just a normal object
+config.accounts.admin.name = "Ima Admin";
+c.set('accounts.admin.id', 1);
+// the config object is returned from require()
+let account123 = c.require('accounts.account123');
+account123.id = 123;
+account123.name = "Account 123";
+```
+
+Computed config values using the `setComputed()` API add on a much more
+powerful feature where values will be calculated at runtime.
+
+```js
+// get the Config wrapper for the default bedrock.config
+var c = bedrock.util.config;
+// set static values
+c.set('server.port', 8443);
+c.set('server.domain', 'bedrock.dev');
+// set a computed value that uses values from the main config
+c.setComputed('server.host', () => {
+  return config.server.domain + ':' + config.server.port;
+});
+console.log(config.server.host); // "bedrock.dev:8443"
+```
+
+The logic for a computed value can be any normal code.
+
+```js
+c.setComputed('server.host', () => {
+  // only add the port if it's not the well known default
+  if(config.server.port !== 443) {
+    return config.server.domain + ':' + config.server.port;
+  }
+  return config.server.domain;
+});
+```
+
+`setComputed()` can be verbose. a wrapper can be created using the standard
+`bind()` functionality. A helpper called `computer()` will do this for you.
+```js
+let cc = bedrock.config.computer();
+cc('server.host', () => {
+  // only add the port if it's not the well known default
+  if(config.server.port !== 443) {
+    return config.server.domain + ':' + config.server.port;
+  }
+  return config.server.domain;
+});
+```
+
+Copmuted values can also be created with string templates.
```

Comment:
Maybe a link to docs would be helpful.  If we use this syntax, it appears that the lodash website will redirect to the latest API version: `https://lodash.com/docs/#template`